app/Repositories/PhotoRepository.php

```diff
@@ -83,20 +84,30 @@ public function create(array $data, $signupId)
      */
     public function update($signup, $data)
     {
-        // @TODO: remove the below logic when we are no longer supporting the phoenix-ashes campaign template.
-        // Update the signup's quantity and why_participated data.
+
+        // Update the signup's quantity and why_participated data and log as an event.
         // We will always update these since we can't tell if this has been changed in a good way yet.
-        $data['quantity_pending'] = $data['quantity'];
+        // @TODO: remove the below logic when we are no longer supporting the phoenix-ashes campaign template.
+        // @TODO: separate event_type into update_why and update_quantity.
+        $updateSignupData = [];
+        $updateSignupData['northstar_id'] = $data['northstar_id'];
+        $updateSignupData['event_type'] = 'update_signup';
+        $updateSignupData['submission_type'] = $data['submission_type'];
+        $updateSignupData['quantity_pending'] = $data['quantity'];
+        $updateSignupData['why_participated'] = $data['why_participated'];
+        $updateSignupData['source'] = $data['source'];
+        $updateSignupData['remote_addr'] = $data['remote_addr'];
```

Comment:
I like something similar here using possibly this: https://laravel.com/docs/5.2/helpers#method-array-only
`$updateSignupData = array_only($data, ['northstar_id', 'event_type', ...]);`?